web/src/components/core/IssuesDialog.jsx

```diff
@@ -78,39 +77,13 @@ const IssueItems = ({ issues = [] }) => {
  *
  * @param {object} props
  * @param {import ("~/client/issues").ClientsIssues} props.issues
+ * @param {string} props.sectionId - A string which indicites which issues section should be highlighted.
  */
 const IssuesSections = ({ issues }) => {
-  const productIssues = issues.product || [];
-  const storageIssues = issues.storage || [];
-  const softwareIssues = issues.software || [];
-
   return (
-    <>
-      <If
-        condition={productIssues.length > 0}
-        then={
-          <Section key="product-issues" title={_("Product")} icon="inventory_2">
-            <IssueItems issues={productIssues} />
-          </Section>
-        }
-      />
-      <If
-        condition={storageIssues.length > 0}
-        then={
-          <Section key="storage-issues" title={_("Storage")} icon="hard_drive">
-            <IssueItems issues={storageIssues} />
-          </Section>
-        }
-      />
-      <If
-        condition={softwareIssues.length > 0}
-        then={
-          <Section key="software-issues" title={_("Software")} icon="apps">
-            <IssueItems issues={softwareIssues} />
-          </Section>
-        }
-      />
-    </>
+    <Section aria-label="issues-section">
```

Comment:
As commented before, maybe we can go for a `_("List of issues")_` or `_("List of found issues")_` instead. Something more "human readable". If you update it, do not forget about the `// TRANSLATORS:` comment to help them to understand the context of the text.